src/scantailor/app/ui/panels/new_open_project.py

```
@@ -0,0 +1,157 @@
+        # Add buttons for recent projects
+        for project_path in self._recent_projects[: self.MAX_RECENT_PROJECTS]:
+            btn = QtWidgets.QPushButton(project_path.name)
+            btn.setToolTip(str(project_path))
+            btn.setFlat(True)
+            btn.setStyleSheet(
+                "QPushButton { text-align: left; padding: 4px; } "
+                "QPushButton:hover { background: palette(highlight); "
+                "color: palette(highlighted-text); }"
+            )
+            btn.clicked.connect(
+                lambda checked, p=project_path: self.recent_project_selected.emit(p)
```

Ruff (0.15.0)

[warning] Unused lambda argument: `checked`

(ARG005)